### PR TITLE
Initial Load

### PR DESCRIPTION
**Pull request by torstenboettjer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
-This project is open source.  Please submit your contributions by forking this repository and submitting a pull request!  Oracle appreciates any contributions that are made by the open source community.
-
-## License
-Copyright (c) 2022 Oracle and/or its affiliates.
-
-Licensed under the Universal Permissive License (UPL), Version 1.0.
-
-See [LICENSE](LICENSE) for more details.
-
-ORACLE AND ITS AFFILIATES DO NOT PROVIDE ANY WARRANTY WHATSOEVER, EXPRESS OR IMPLIED, FOR ANY SOFTWARE, MATERIAL OR CONTENT OF ANY KIND CONTAINED OR PRODUCED WITHIN THIS REPOSITORY, AND IN PARTICULAR SPECIFICALLY DISCLAIM ANY AND ALL IMPLIED WARRANTIES OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY, AND FITNESS FOR A PARTICULAR PURPOSE.  FURTHERMORE, ORACLE AND ITS AFFILIATES DO NOT REPRESENT THAT ANY CUSTOMARY SECURITY REVIEW HAS BEEN PERFORMED WITH RESPECT TO ANY SOFTWARE, MATERIAL OR CONTENT CONTAINED OR PRODUCED WITHIN THIS REPOSITORY. IN ADDITION, AND WITHOUT LIMITING THE FOREGOING, THIRD PARTIES MAY HAVE POSTED SOFTWARE, MATERIAL OR CONTENT TO THIS REPOSITORY WITHOUT ANY REVIEW. USE AT YOUR OWN RISK.
```
Comment (VetteByte) [inline README.md]: This should remain in the README.  Please add back.

**Review by VetteByte — CHANGES_REQUESTED** [review]
Everything looks good except for the README (please add back the license/disclaimer section).

File: README.md (hunk 1 above)
Comment (torstenboettjer) [reply README.md]: Done

**Review by torstenboettjer — COMMENTED** [review]
Done

**Review by VetteByte — APPROVED** [review]
Great work!

**Pull request merged**